opcUaDevSupApp/drvOpcUa.cpp

```diff
@@ -801,7 +801,9 @@ long OpcWriteValue(int opcUaItemIndex,double val,int verbose)
     return 0;
 }
 /* iocShell: record write func  */
+extern "C" {
```

Comment:
I don't think these are necessary.
`epicsExport.h` contains these declarations (has been for a long time).